### DISCUSSION TITLE
Mojang登录问题

### DISCUSSION THREAD

**Discussion by NOTAANAMEE** [Meloong-Git/PCL#3994] [提问] [answered]
Mojang的服务器是寄了吗？登录时偶尔会传回HTML并触发报错。
不是PCL2的问题，自己写的也会出现这种情况
网络状况良好，不存在被墙的情况
![屏幕截图 2024-06-09 160528](https://github.com/Hex-Dragon/PCL2/assets/160555024/4a8a0dd6-e936-4063-891c-208a1e6b2207)

**Comment by NOTAANAMEE** [comment]
今天第一次打开PCL2出现这种情况了

**Comment by MoYuan-CN** [comment, marked as answer]
回传了 429……
这说明你请求服务器请求的太快太多了
